### PR TITLE
added new files

### PR DESCRIPTION
**Pull request by IgorPushnov**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
-const textExample = "Hello world Hello all";
+const fs = require('fs');
+
+const array = fs.readFileSync('util.js').toString().split(" ");
+// for (i in array) {
+//   console.log(array[i]);
+// }
+
+const textExample = array.join(" ").replace(/\<+.+?(?=\>)|\>|\/|\W|\s/g, " ").replace(/\s+/g, " ");
```
Comment (shnax0210) [inline main.js]: Please move all word parsing logic to `parseWords` function.

File: dog.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+h1 {
```
Comment (shnax0210) [inline dog.css]: Please move all test files to separate folder with appropriate name, something like "testData".

File: main.js (hunk 1 above)
Comment (IgorPushnov) [reply main.js]: changes made

File: dog.css (hunk 2 above)
Comment (IgorPushnov) [reply dog.css]: changes made

**Pull request merged**